### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by 0xxlegolas**

### CODE REVIEW

File: item-seller/packages/contracts/.env [hunk 1]
```diff
@@ -20,11 +20,14 @@ CHAIN_ID=31337
...
 #SMART STORAGE UNIT ID
+#Copy this info from in game smart storage unit
 SSU_ID=06496282436511424955311689125460433858625928370621647559384988961918634545666
 
+
 ##### ITEM SELLER CONFIGURATION
-#ITEM ID 77800 - Common Ore
-INVENTORY_ITEM_ID=1235
+# Its a prerequisite to have the item deposited into the SSU
+# Once deposited get the itemId from https://blockchain-gateway-nova.nursery.reitnorf.com/smartassemblies/<ssu_id>
```
Comment (rhubarbgarden) [inline item-seller/packages/contracts/.env]: Needs clearer explanation - users should get the `itemId` from the corresponding item in the `inventory.storageItems` array of their SSU at that address.

File: item-trade/packages/contracts/.env [hunk 2]
```diff
@@ -17,12 +17,15 @@ RPC_URL="http://127.0.0.1:8545"
 CHAIN_ID=31337
 
 #SMART STORAGE UNIT ID
+#Copy this info from in game smart storage unit
 SSU_ID=11196282436511424955311689125460433858625928370621647559384988961918634545666
 
+# Its a prerequisite to have the item deposited into the SSU
+# Once deposited get the itemId from https://blockchain-gateway-nova.nursery.reitnorf.com/smartassemblies/<ssu_id>
```
Comment (rhubarbgarden) [inline item-trade/packages/contracts/.env]: Needs clearer explanation - users should get the itemId from the corresponding item in the inventory.storageItems array of their SSU at that address.

**Review by rhubarbgarden — APPROVED** [review]
Some improvements could be made with commenting how to access the itemId. These are really helpful improvements

File: item-seller/packages/contracts/.env (hunk 1 above)
Comment (0xxlegolas) [reply item-seller/packages/contracts/.env]: Yes adding that along with the screenshots in docs

**Review by CCP-Red-Dragon — APPROVED** [review]

**Pull request merged**